modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -59,22 +59,29 @@ def perform(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
         start_docker_container_job(auto_claim&.id) if auto_claim.status != errored_state_value
       rescue Faraday::Error::ParsingError, Faraday::TimeoutError => e
         set_errored_state_on_claim(auto_claim)
+        error_message = get_error_message(e)
+        error_status = get_error_status_code(error)
+
         log_job_progress(LOG_TAG,
                          claim_id,
-                         "526EZ PDF generator faraday errored #{e.status_code} #{e.original_body}")
+                         "526EZ PDF generator faraday errored #{error_status} #{error_message}")
         log_exception_to_sentry(e)
 
         raise e
       rescue ::Common::Exceptions::BackendServiceException => e
         set_errored_state_on_claim(auto_claim)
+        error_message = get_error_message(e)
+        error_status = get_error_status_code(error)
```

Comment:
`error` is undefined here - I presume `e` is what ya want here.

Reply:
Yep fixing those up now